CMakeLists.txt

```diff
@@ -282,7 +282,7 @@ SET(wasm_dis_SOURCES
 )
 ADD_EXECUTABLE(wasm-dis
                ${wasm_dis_SOURCES})
-TARGET_LINK_LIBRARIES(wasm-dis passes wasm asmjs ir cfg support)
+TARGET_LINK_LIBRARIES(wasm-dis wasm asmjs emscripten-optimizer passes ir cfg support wasm)
```

Comment:
Nit: You'd probably want to revert this before committing, given that there's another patch fix this out there

Reply:
No, this particular build change is necessary because I added a `Pass` to binary-writer.cpp in this PR.